udata/harvest/backends/dcat.py

```diff
@@ -105,13 +110,14 @@ def get_format(self):
                 raise ValueError(msg)
         return fmt
 
-    def parse_graph(self, url, fmt) -> List[Graph]:
+    def walk_graph(self, url: str, fmt: str) -> Generator[tuple[int, Graph], None, None]:
         """
-        Returns an instance of rdflib.Graph for each detected page
-        The index in the list is the page number
+        Process the graphs by executing the `do()` callback on each page.
```

Comment:
I think you need to update this docstring :) The `do()` is also mentioned later on

Reply:
Simplified it in 12aa7943a